pkg/controllers/v1alpha1/mpi_job_controller_test.go

```diff
@@ -549,8 +551,10 @@ func TestLauncherFailed(t *testing.T) {
 	mpiJob := newMPIJob("test", int32Ptr(64), &startTime, nil)
 	f.setUpMPIJob(mpiJob)
 
+	mpiJob.Status.LauncherStatus = kubeflow.LauncherFailed
```

Comment:
what's the usage of this line? mpiJobCopy has the same operation

Reply:
I will delete the line , my bad.